src/test/groovy/nebula/plugin/dependencyverifier/DependencyResolutionVerifierTest.groovy

```diff
@@ -1177,7 +1177,7 @@ class DependencyResolutionVerifierTest extends IntegrationTestKitSpec {
     private static String taskThatRequiresConfigurationDependencies() {
         return """
             task taskWithConfigurationDependencies {
-                inputs.files configurations.compileClasspath.incoming.artifacts
+                inputs.files configurations.compileClasspath
```

Comment:
@OdysseusLives I think for the test purpose this should be fine but maybe take a look if I'm missing anything.

Reply:
@chali this test is making sure we have this spot at [project.gradle.buildFinished](https://github.com/nebula-plugins/gradle-dependency-lock-plugin/blob/master/src/main/kotlin/nebula/plugin/dependencyverifier/DependencyResolutionVerifier.kt#L75-L83) which collects dependency resolution errors before tasks were executed/ the tasks themselves were mis-configured

rather than only using [project.gradle.taskGraph.whenReady](https://github.com/nebula-plugins/gradle-dependency-lock-plugin/blob/master/src/main/kotlin/nebula/plugin/dependencyverifier/DependencyResolutionVerifier.kt#L85-L112) which collects dependency resolution after tasks have been executed

so if this can be adjusted to fail for task configuration rather than execution, that would be ideal